test/test_onnx.py

```diff
@@ -35,6 +36,17 @@ def run_model(self, model, inputs):
         # validate the exported model with onnx runtime
         self.ort_validate(onnx_io, inputs, outputs)
 
+        # verify with the other inputs
+        if other_inputs:
```

Comment:
I thought it would be clearer to separate the input used to export the model.
But a single list would be fine too. 

Will make the change in the next commit.